spec.html

```diff
@@ -4178,8 +4178,8 @@ <h1>The Reference Record Specification Type</h1>
           </tr>
           <tr>
             <td oldids="sec-getreferencedname,ao-getreferencedname">[[ReferencedName]]</td>
-            <td>a String, a Symbol, or a Private Name</td>
-            <td>The name of the binding. Always a String if [[Base]] value is an Environment Record.</td>
+            <td>a String, a Symbol, a Private Name, or an ECMAScript language value</td>
```

Comment:
```suggestion
            <td>an ECMAScript language value or a Private Name</td>
```
since ECMAScript language value subsumes String and Symbol

Reply:
That makes sense; hopefully the description suffices to clarify.